### ISSUE TITLE
RedisConnection.set([B[B)V

### ISSUE THREAD

**Issue by hthhit** [spring-projects/spring-security-oauth2-boot#20]
nested exception is java.lang.NoSuchMethodError: org.springframework.data.redis.connection.RedisConnection.set([B[B)V

**Comment by jgrandja** [comment]
@hthhit What version of `spring-security-oauth2` are you using? This issue has been fixed in **2.3.2.RELEASE**.

**Comment by MadeInChina** [comment]
same issue 

```<dependency>
        <!-- Import dependency management from Spring Boot -->
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-dependencies</artifactId>
        <version>2.0.2.RELEASE</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
```
Spring boot dependencies 2.0.2.RELEASE are using 

spring-security-oauth2-autoconfigure 2.0.0.RELEASE

```
[INFO] +- org.springframework.security.oauth.boot:spring-security-oauth2-autoconfigure:jar:2.0.0.RELEASE:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.0:compile
[INFO] |  +- org.springframework.boot:spring-boot:jar:2.0.2.RELEASE:compile
[INFO] |  +- org.springframework.boot:spring-boot-autoconfigure:jar:2.0.2.RELEASE:compile
[INFO] |  +- org.springframework.security.oauth:spring-security-oauth2:jar:2.2.1.RELEASE:compile
```

so need upgrade spring-security-oauth2:jar:2.2.1.RELEASE to 2.3.2.RELEASE

**Comment by jgrandja** [comment]
@hthhit Closing this issue as this has been resolved in the 2.3.2.RELEASE. Please update to use this dependency `org.springframework.security.oauth:spring-security-oauth2:jar:2.3.2.RELEASE`
